### PR TITLE
RxFire: Api Change and documentation

### PR DESCRIPTION
**Pull request by davideast**
**Changes**:
* Per user feedback `docChanges()` has been named to `collectionChanges()`.
* Documentation for Storage

### CODE REVIEW

File: packages/rxfire/docs/storage.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+import * as firebase from 'firebase';
+import 'firebase/storage';
+
+// Set up Firebase
+const app = initializeApp({ /* config */ });
+const storage = app.storage();
+const davidRef = db.ref('users/david.png');
+
+// Upload a transparent 1x1 pixel image
+const task = davidDoc.putString('R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7', 'base64');
```
Comment (hiranya911) [inline packages/rxfire/docs/storage.md]: Should this be `davidRef`?

File: packages/rxfire/docs/storage.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+#### TypeScript Example
+```ts
+import { percentage } from 'rxfire/firestore';
+import * as firebase from 'firebase';
+import 'firebase/storage';
+
+// Set up Firebase
+const app = initializeApp({ /* config */ });
+const storage = app.storage();
+const davidRef = db.ref('users/david.png');
```
Comment (hiranya911) [inline packages/rxfire/docs/storage.md]: `storage.ref()`?

File: packages/rxfire/docs/storage.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+#### TypeScript Example
+```ts
+import { fromTask } from 'rxfire/firestore';
+import * as firebase from 'firebase';
+import 'firebase/storage';
+
+// Set up Firebase
+const app = initializeApp({ /* config */ });
+const storage = app.storage();
+const davidRef = db.ref('users/david.png');
```
Comment (hiranya911) [inline packages/rxfire/docs/storage.md]: `storage.ref()`?

File: packages/rxfire/docs/storage.md (hunk 1 above)
Comment (davideast) [reply packages/rxfire/docs/storage.md]: fixed!

File: packages/rxfire/docs/storage.md (hunk 2 above)
Comment (davideast) [reply packages/rxfire/docs/storage.md]: fixed!

File: packages/rxfire/docs/storage.md (hunk 3 above)
Comment (davideast) [reply packages/rxfire/docs/storage.md]: fixed!

**Pull request merged**